lt/cmd/speech/main.go

```diff
@@ -11,17 +15,43 @@ import (
 	"github.com/mattermost/mattermost-plugin-calls/lt/client"
 )
 
+const (
+	duration = 10 * time.Minute
+)
+
+var script, siteURL, wsURL, channelID, teamID, userPassword, profile string
+var setup bool
+
 func main() {
+	flag.StringVar(&script, "script", "script.txt", "Script for the tts")
```

Comment:
This is not working by default because it's looking in the current directory. I'd probably move the scripts to a dedicated folder, maybe on the root of the load-tester.